Sources/Sentry/SentryTracer.m

```diff
@@ -200,6 +211,8 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
 #endif // SENTRY_HAS_UIKIT
 
 #if SENTRY_TARGET_PROFILING_SUPPORTED
+        // TODO(ref): Move Profiling to a SentryTracerExtension.
+        // https://github.com/getsentry/sentry-cocoa/issues/2736
```

Comment:
You can highlight lines of code in your diff and copy the URL from your browser address bar, then put them in a bullet point list in the description of that issue or in a separate comment. Eg https://github.com/getsentry/sentry-cocoa/pull/2724/files#diff-84fda3cf94bf21dd7ceabf3c0ced60d74c124620d5a8dfdd3b8661563a79ec49R216

Reply:
Got it. My problem with this idea, as mentioned in [this](https://github.com/getsentry/sentry-cocoa/pull/2724#discussion_r1119863773) thread, is that those issues are never tackled. 

If we have a reminder in the code, when someone starts to work on a new feature that needs to change one of this TODOs places, they could also include this change. 

Lets say we have another feature that also requires `Finish callback` from SentryTracer. Instead of making use of the current property, or even worse, create another one, since it only accepts one callback, we would see the TODO and remember this could be done with a SentryTracerExtension.

Reply:
Ah, sorry I missed that thread. I don't feel terribly strongly about this, I'm guilty of it myself.